### PR TITLE
Add release notes for 2.8.0

### PR DESCRIPTION
**Pull request by serinamarie**
[Rendered](https://github.com/PrefectHQ/prefect/blob/release-notes-2.8.0-updated/RELEASE-NOTES.md#release-280)

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Prefect Release Notes
 
+## Release 2.8.0
+
+### Work Pools 🎉
+- Remove experimental designation around work pools — https://github.com/PrefectHQ/prefect/pull/8362
+- Adds work pool and queue filters to flow runs history endpoint — https://github.com/PrefectHQ/prefect/pull/8459
+- Update flowRunFilter on WorkPool page — https://github.com/PrefectHQ/prefect/pull/8453
+- Updates agent logic to respect work queue priority — https://github.com/PrefectHQ/prefect/pull/8458
```
Comment (serinamarie) [inline RELEASE-NOTES.md:9]: To be updated

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+
+
+### Enhancements
+- Add ability to create a new flow run from the UI with custom default parameter values taken from a previous run — https://github.com/PrefectHQ/prefect/pull/8405
```
Comment (madkinsz) [inline RELEASE-NOTES.md:13]: Maybe a more succinct name? Not a strong preference
```suggestion
- Add ability to create a flow run from the UI with parameters from a previous run — https://github.com/PrefectHQ/prefect/pull/8405
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+- Add generic `Webhook` block — https://github.com/PrefectHQ/prefect/pull/8401
+- Add override customizations functionality to deployments via CLI — https://github.com/PrefectHQ/prefect/pull/8349
+- Add ability to reset concurrency limits in CLI to purge existing runs from taking concurrency slots — https://github.com/PrefectHQ/prefect/pull/8408
+- Ensure matching flow run state information in UI — https://github.com/PrefectHQ/prefect/pull/8441
+- Customize CLI block registration experience based on `PREFECT_UI_URL` — https://github.com/PrefectHQ/prefect/pull/8438
+
+### Fixes
+- Fix `prefect dev start` command — https://github.com/PrefectHQ/prefect/pull/8176
+- Fix ability to see long log messages when in Prefect UI — https://github.com/PrefectHQ/prefect/pull/8449
```
Comment (madkinsz) [inline RELEASE-NOTES.md:22]:
```suggestion
- Fix display of long log messages when in the UI — https://github.com/PrefectHQ/prefect/pull/8449
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+- Update `get_run_logger` to accomodate returning `logging.LoggerAdapter` — https://github.com/PrefectHQ/prefect/pull/8422
+- Restore Prefect wrapper around HTTP errors for nicer error messages — https://github.com/PrefectHQ/prefect/pull/8391
+
+### Documentation
+- Update Infrastructure concept documentation with extra-pip-package example and updated deployment.yaml — https://github.com/PrefectHQ/prefect/pull/8465
+- Ensure Prefect docs UI components compatible with Material for Mkdocs 9.0.10 — https://github.com/PrefectHQ/prefect/pull/8403
```
Comment (madkinsz) [inline RELEASE-NOTES.md:28]: Probably a `maintenance` item and could be excluded.

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,77 @@
...
 
+## Release 2.8.0
+
+### Prioritize flow runs with work pools 🏊
+
+![Work pools allow you to organize and prioritize work](https://user-images.githubusercontent.com/12350579/217914094-e8064420-294b-4033-b12e-c0f58da521d5.png)
+
+With this release, flow runs can now be prioritized between work queues via work pools! Work pools allow you to organize and prioritize work by grouping related work queues together. Within work pools, you can assign a priority to each queue, and flow runs scheduled on higher priority work queue will be run before flow runs scheduled on lower priority work queues. This allows agents to prioritize work that is more important or time-sensitive even if there is a large backlog of flow runs on other work queues in a given work pool.
+
+All existing work queues will be assigned to a default work pool named `default-agent-pool`. Creating a new work pool can be done via the Work Pool tab in the UI or the CLI.
```
Comment (tpdorsey) [inline RELEASE-NOTES.md:11]:
```suggestion
All existing work queues will be assigned to a default work pool named `default-agent-pool`. Creating a new work pool can be done via the Work Pools page in the UI or the CLI.
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,77 @@
...
+```
+
+Starting an agent with the `--pool` command will allow the agent to pick up flow runs for the entire pool even as new queues are added to the pool. If you want to start an agent that only picks up flow runs for a specific queue, you can use the `--queue` flag.
+
+```bash
+prefect agent start --pool my-pool --queue high-priority
+```
+
+To learn more about work pools, check out the [docs](https://docs.prefect.io/concepts/work-pools/) or see the relevant pull requests:
+- Add ability to filter on work pool and queue when viewing flow run history — https://github.com/PrefectHQ/prefect/pull/8459
```
Comment (tpdorsey) [inline RELEASE-NOTES.md]:
```suggestion

- Add ability to filter on work pool and queue when viewing flow run history — https://github.com/PrefectHQ/prefect/pull/8459
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,77 @@
...
+### Fixes
+- Fix `prefect dev start` command — https://github.com/PrefectHQ/prefect/pull/8176
+- Fix display of long log messages when in the UI — https://github.com/PrefectHQ/prefect/pull/8449
+- Update `get_run_logger` to accomodate returning `logging.LoggerAdapter` — https://github.com/PrefectHQ/prefect/pull/8422
+- Restore Prefect wrapper around HTTP errors for nicer error messages — https://github.com/PrefectHQ/prefect/pull/8391
+- Fix display of work pool flow run filter in the UI — https://github.com/PrefectHQ/prefect/pull/8453
+
+### Documentation
+- Update Infrastructure concept documentation with `extra-pip-package` example and updated `deployment.yaml` — https://github.com/PrefectHQ/prefect/pull/8465
+
```
Comment (tpdorsey) [inline RELEASE-NOTES.md:71]:
```suggestion
- Add Work Pools documentation - https://github.com/PrefectHQ/prefect/pull/8377

```

Comment (serinamarie) [reply RELEASE-NOTES.md:71]: thanks!

**Review by madkinsz — APPROVED** [review]

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,79 @@
...
+```bash
+prefect agent start --pool my-pool --queue high-priority
+```
+
+To learn more about work pools, check out the [docs](https://docs.prefect.io/concepts/work-pools/) or see the relevant pull requests:
+
+- Add ability to filter on work pool and queue when querying flow runs — https://github.com/PrefectHQ/prefect/pull/8459
+- Remove experimental designation around work pools — https://github.com/PrefectHQ/prefect/pull/8362
+- Ensure agent respects work queue priority — https://github.com/PrefectHQ/prefect/pull/8458
```
Comment (madkinsz) [inline RELEASE-NOTES.md:53]: These can be listed in the normal `enhancements` section.

Comment (serinamarie) [reply RELEASE-NOTES.md:53]: I merged right after approval but added those changes here:
https://github.com/PrefectHQ/prefect/pull/8474

**Pull request merged**